src/app.jsx

```diff
@@ -110,18 +113,31 @@ export const Application = () => {
 
     const visibleFiles = files.filter(file => !file.name.startsWith("."));
 
+    const contextMenuItems = (
+        <MenuList>
+            <MenuItem className="contextMenuOption" onClick={() => { createDirectory(Dialogs, "/" + path.join("/") + "/") }}>
```

Comment:
Can we please use only lower case for class names? for seperating words in class names feel free to use dashes.